common/services/prismic/api.js

```diff
@@ -16,7 +17,12 @@ let memoizedPrismic;
 
 function periodicallyUpdatePrismic() {
   setInterval(async () => {
-    memoizedPrismic = await Prismic.getApi(apiUri);
+    try {
+      memoizedPrismic = await Prismic.getApi(apiUri);
+    } catch (error) {
+      // TODO: maybe remove once we've established the error(s)
```

Comment:
Nope, we should leave this here as if it's erroring,. we should know.

Reply:
Is there a way to limit its impact on our Sentry quota?